packages/zoe/src/contractFacet/zcfSeat.js

```diff
@@ -214,6 +216,10 @@ export const createSeatManager = (
 
         return isOfferSafe(state.proposal, reallocation);
       },
+      /**
+       * @deprecated switch to zcf.atomicRearrange()
```

Comment:
This is good but note that it consumers of this object aren't getting the type written here. They get `ZCFSeat` which has the problem discussed above.

Reply:
Agreed. The comment is mostly for those reading/editing this file. This still seemed the most concise way to express it.